comp/checks/windowseventlog/windowseventlogimpl/check/config_test.go

```diff
@@ -192,9 +193,9 @@ func TestInvalidRegexp(t *testing.T) {
 		errorMatch string
 	}{
 		{"lookahead", "(?=foo)", "invalid or unsupported Perl syntax: `(?=`"},
-		{"lookbehind", "(?<=foo)", "invalid or unsupported Perl syntax: `(?<`"},
+		{"lookbehind", "(?<=foo)", "invalid named capture: `(?<="},
```

Comment:
Can you gives context about this change? Did the error message changed?

Reply:
Yes. The error messages changed and the test broke. I updated the test so CI stops complaining